packages/toolpad-core/src/SignInPage/SignInPage.tsx

```diff
@@ -40,6 +39,29 @@ import FusionAuthIcon from './icons/FusionAuth';
 import { BrandingContext, RouterContext } from '../shared/context';
 import { DocsContext } from '../internal/context';
```

Comment:
This looks like a red flag (making the logic depend on something related to the docs), unless I am misunderstanding the usage.

Reply:
This was done to help render demos smoothly in the docs. It's used in two places, mostly to do with UI:

- Not setting `autoFocus` true if the component is being used in the docs
- Setting the loading state of the Credentials form to `false` even if the login is unsuccessful (also the case in the docs)

Agreed that it's not the cleanest way - I can make a note to remove these and tackle that in one of the upcoming releases 👍🏻